src/components/volunteer-schedule/ScheduleTab/ScheduleTab.tsx

```diff
@@ -0,0 +1,56 @@
+import VolunteerEventList from '@/components/volunteer-schedule/VolunteerEventList/VolunteerEventList';
+import { volunteerEventsMock } from '@/components/volunteer-schedule/mock';
+import * as styles from './ScheduleTab.css';
+import DangleCalendar from '@/components/common/Calendar/DangleCalendar';
+import { useCallback, useMemo, useState } from 'react';
+import moment, { Moment } from 'moment';
+interface ScheduleTabProps {}
+
+const ScheduleTab: React.FC<ScheduleTabProps> = ({}) => {
+  const [volunteerEvents, setVolunteerEvents] = useState(volunteerEventsMock);
+  const [focusedDate, setFocusedDate] = useState<string>();
+
+  const eventDates = useMemo(
+    () => volunteerEvents.map(e => e.startAt),
+    [volunteerEvents]
+  );
```

Comment:
```suggestion
  const eventDates = useMemo(
    () => volunteerEvents.map(e => moment(e.startAt).format('YYYY-MM-DD')),
    [volunteerEvents]
  );
```
이 부분 moment로 한번 변환해줘야 캘린더에 닷아이콘이 렌더링 되네요!!

Reply:
오 놓쳤던 부분 알려주셔서 감사합니다!